cfgov/v1/__init__.py

```diff
@@ -93,6 +98,7 @@ def environment(**options):
         'localtime': template_localtime,
         'pluralize': pluralize,
         'slugify': slugify,
+        'dateformatter': date_formatter
```

Comment:
Trivial: place `dateformatter` at the top of this dictionary, above `linebreaksbr`, so it'll retain alphabetical order.